arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3474,10 +3474,57 @@
           "type": "string",
           "readOnly": true,
           "description": "Gets the workflow trigger callback URL."
+        },
+        "method": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the workflow trigger callback URL HTTP method."
+        },
+        "basePath": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the workflow trigger callback URL base path."
+        },
+        "relativePath": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the workflow trigger callback URL relative path."
+        },
+        "relativePathParameters": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets the workflow trigger callback URL relative path parameters."
+        },
+        "queries": {
+          "$ref": "#/definitions/WorkflowTriggerListCallbackUrlQueries",
+          "description": "Gets the workflow trigger callback URL query parameters."
         }
       },
       "description": "The workflow trigger callback URL."
     },
+    "WorkflowTriggerListCallbackUrlQueries": {
```

Comment:
this is missing a description. this adds a new error to the list of existing errors in the spec. could you add a description please?